pages/auth/login.tsx

```diff
@@ -64,7 +64,7 @@ const Login: NextPageWithLayout<
     },
     validationSchema: Yup.object().shape({
       email: Yup.string().required().email(),
-      password: Yup.string().required(),
+      password: Yup.string().required().min(8),
```

Comment:
@Dhoni77 Please don't do min password length check during login. It's not a good practice.

Reply:
done